src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, slug, sex } = person;
+
+  return (
+    <tr data-cy="person">
+      <td>
+        <Link
+          to={`/people/${slug}`}
+          className={sex === 'f' ? 'has-text-danger' : ''}
+        >
+          {name}
+        </Link>
+      </td>
+    </tr>
+  );
```

Comment:
The `PersonLink` component returns a `<tr>` element, which is not appropriate unless it is used within a `<table>` structure. Consider returning a `<Link>` component directly without wrapping it in a `<tr>` and `<td>`, or ensure this component is used within a `<table>` context.